driver/syscall_table.c

```diff
@@ -386,6 +386,9 @@ const enum ppm_syscall_code g_syscall_code_routing_table[SYSCALL_TABLE_SIZE] = {
 #ifdef __NR_select
 	[__NR_select - SYSCALL_TABLE_ID0] = PPM_SC_SELECT,
 #endif
+#ifdef __NR_time
+	[__NR_time - SYSCALL_TABLE_ID0] = PPM_SC_TIME,
+#endif
```

Comment:
I see this definition and line 291 of [the same file](https://github.com/linux-on-ibm-z/sysdig/blob/9c2fb7f33ef9a6390d045373bae537d9768b700f/driver/syscall_table.c#L291). Why you need to add it? You can ifdef that one in case.